File: articles/metrics-objectives-and-goodharts-law/c/goodhart_risk.c
#include <stdio.h>

int main(void) {
    double proxy_gap = 0.38;
    double pressure = 0.88;
    double gaming = 0.72;
    double guardrail_penalty = 1.0;
    double score = (proxy_gap + pressure + gaming + guardrail_penalty) / 4.0;
    printf("goodhart_risk_score=%.4f\n", score);
    return 0;
}

File: articles/metrics-objectives-and-goodharts-law/rust/goodhart_risk.rs
fn main() {
    let proxy_gap = 0.38;
    let pressure = 0.88;
    let gaming = 0.72;
    let guardrail_penalty = 1.0;
    let score = (proxy_gap + pressure + gaming + guardrail_penalty) / 4.0;
    println!("goodhart_risk_score={:.4}", score);
}
